src/pages/Home.js

```diff
@@ -25,8 +25,17 @@ export default function Home() {
         <LoginSection />
         <hr />
         <StyledGuestList>
-          {guestArray.map((guest, index) => {
-            return <GuestCard key={index} personalData={guest} />;
+          {guestArray.map((guest) => {
+            //when you create a new guest, there is no guest.id that you could use as a key
+            //every guest gets an id when you save
+            //the guestArray to the dataBase (to prevent one guest having two ids)
+            //I know Math.random() can't be the correct solution here
+            return (
+              <GuestCard
+                key={guest._id ?? Math.random()}
```

Comment:
```suggestion
                key={guest._id ?? Math.random().toString(36).substring(2)}
```
That's not a bad idea! Here's a slight improvement. Ids are usually a mix of numbers and characters. In JS you can convert a given number to a string representation. The number within the `toString` method converts it to the respective number system: 10 would be the decimal system, 2 would be the binary system, and 36 is ... I don't know what that system is called 🤣. But it's a perfect mix for the purpose of ids.

Reply:
Thanks! I've read your suggestion with mapping the guestArray to itself just now, so it was a little too late...
But if this solution here is ok, I implement your suggestion and just go with it 👍